build.gradle.kts

```diff
@@ -43,35 +41,28 @@ dependencies {
     testImplementation("org.mock-server:mockserver-netty:5.10.0")
```

Comment:
Can you bump this to 5.11.1 to see if the snyk issue is mitigated?

Reply:
It shouldn't fix the issue, since it's a licensing mismatch, not a security issue.

5.10.0 has this path: org.mock-server:mockserver-netty@5.10.0 > org.mock-server:mockserver-core@5.10.0 > com.github.java-json-tools:json-schema-validator@2.2.11 > com.github.java-json-tools:json-schema-core@1.2.10 > org.mozilla:rhino@1.7.7.1 

5.11.1 has this path: org.mock-server:mockserver-netty@5.11.1 > org.mock-server:mockserver-core@5.11.1 > com.github.java-json-tools:json-schema-validator@2.2.14 > com.github.java-json-tools:json-schema-core@1.2.14 > org.mozilla:rhino@1.7.7.2

Both rhino versions have MPL-2 per [mvnrepository.com](https://mvnrepository.com/artifact/org.mozilla/rhino/1.7.7.2).